internal/util/vault_sa.go

```diff
@@ -222,6 +237,7 @@ func isTenantSAConfigOption(opt string) bool {
 	// additional options for VaultTenantSA
 	switch opt {
 	case "tenantSAName":
+	case "vaultAuthPath":
```

Comment:
@nixpanic do we need to add this configuration to the configmap struct which contains`VAULT_` keys?

Reply:
non of the Vault Tenant SA options are listed there. We could add them, but maybe we need to consider a cleaner approach that allows aliases for different options.